go-controller/hack/test-go.sh

```diff
@@ -23,10 +23,12 @@ function testrun {
     go test -mod vendor ${args}
 }
 
+root_pkgs=("github.com/ovn-org/ovn-kubernetes/go-controller/pkg/node" "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/controller")
+
 i=0
 for pkg in ${PKGS}; do
     # This package requires root
```

Comment:
reword the comment and move it to above the variable declaration?

(also, boo! unit tests shouldn't require being root. grump grump grump.)

Reply:
Fixed.